release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1723,6 +1722,82 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
+
+* Previously, installing a cluster to the AWS Commercial Cloud Services (C2S) `us-iso-east-1` region failed with an error message stating an `UnsupportedOperation`. With this fix, installing to this region now succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-2324[*OCPBUGS-2324*])
+
+* Previously, installations on AWS could fail because the installation program did not create the cloud.conf file with the necessary service endpoints in it. This led to the machine config operator creating an empty cloud.conf file that lacked the service endpoints, leading to an error. With this update, the installation program always creates the cloud.conf file so that the installation succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-20401[*OCPBUGS-20401*])
+
+* Previously, if you installed a cluster using the Agent-based installer and your pull secret had a null "auth" or "email" field, the installation would fail without providing a useful error. With this update, the `openshift-install agent wait-for install-complete` command validates your pull secret and notifies you if there are null fields. (link:https://issues.redhat.com/browse/OCPBUGS-14405[*OCPBUGS-14405*])
+
+* Previously the `create agent-config-template` command printed a line with `INFO` only but no details about whether the command was successful and where the template file was written to. Now, if the command is successful, the command will print `INFO Created Agent Config Template in <path> directory`. (link:https://issues.redhat.com/browse/OCPBUGS-13408[*OCPBUGS-13408*])
+
+* Previously, when a user specified the 'vendor' hint in the `agent-config.yaml` file, the value was being checked against the wrong field so that the hint would not match. With this update, the use of the `vendor` hint correctly selects a disk. (link:https://issues.redhat.com/browse/OCPBUGS-13356[*OCPBUGS-13356*])
+
+* For clusters that run on {rh-openstack} that you upgrade to 4.14 and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in link:https://access.redhat.com/solutions/7013893[OpenShift on OpenStack with Availability Zones: Invalid Compute ServerGroup setup during OpenShift deployment]. (link:https://issues.redhat.com/browse/OCPBUGS-13300[*OCPBUGS-13300*])
```

Comment:
This doesn't seem like a bug fix to me? It seems more like it would go in "notable technical change" or a different section.

Reply:
good catch!